src/transformState.js

```diff
@@ -1,11 +1,31 @@
 'use strict';
 
 /**
- * @param {Object} state
- * @param {Object[]} actions
- */
+   * @param {Object} state'
+   * @param {Object[]} actions
+   */
 function transformState(state, actions) {
   // write code here
+
+  for (let i = 0; i < actions.length; i++) {
+    const oneElement = actions[i];
+
+    if (oneElement.type === 'addProperties') {
```

Comment:
in each block `if` you compare one value, there is better suited here `switch/case`